### ISSUE TITLE
Can't run tests because of "Uncaught Error: Mismatched anonymous define() module"

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5#14]
On 21b0cf9 and ckeditor/ckeditor5-core@237ba98 when running tests I get:

```
require.js:166 Uncaught Error: Mismatched anonymous define() module: function () {
      return (root.sinon = factory());
    }
http://requirejs.org/docs/errors.html#mismatch
bender.js:105 Error: Deferment unlock timeout - please check your plugins
    at client.js:433
```

Do I miss something?

**Comment by fredck** [comment]
That's because of this:
https://github.com/benderjs/benderjs-sinon/issues/5

You should update package.json with the proper version and run npm install there.

**Comment by gregpabian** [comment]
Fixed with 69676542e3ad63ea3999593e50be2f883361f93a.
